### ISSUE TITLE
Retiring projects

### ISSUE THREAD

**Issue by stuartpb** [unusualstudio/unusualstudio-projects#7]
Touches on some thoughts from #1.

Right now, I've held off on adding any Stage 3 (Polished, which essentially means "Done") projects, for a couple reasons:

- They don't share a purpose with the stage 0/1/2 projects in that there's usually not much more to do with them
- Sometimes there is more to do with them (ie. updating their style to a newer formulation, ie. classic Node modules using Nodebacks into ES6 modules using Promises), but I'm not sure if I'm all that interested in doing that
- Sometimes they've become irrelevant (or worse) since reaching stage 3, and I wouldn't want to work on them further
- Of the projects that reach Stage 3, I'd want to sort some of them by dex, and some of them I'd just want to sink based on the stuff I described above

Basically, if I start adding Stage 3 projects, I think I'd need more fields to qualify some of these details, which would matter for their modal presentation (and I might want to figure out a sensible default, for how to interpret a project that otherwise looks like a stage 0/1/2 project with the kinds of fields that are defined right now).

**Comment by stuartpb** [comment]
Some field ideas to encapsulate some of these sentiments: `archived`, `maintenance` (for "maintenance mode")

Could be bools, string enum values, or string enum fields (where maintenance might be, like, 'me', or maybe I've handed the project off to a new maintainer - which is probably a bridge I should cross when I get to it)

**Comment by stuartpb** [comment]
There's a lot of overlap (and even subjectivity as toward the difference) between retiring a project and abandoning it (#10).

**Comment by stuartpb** [comment]
Indeed, is there a difference between a retired project and a project that's abandoned after becoming viable?

**Comment by stuartpb** [comment]
In relation to the `pass` and `disregard` fields from #10, I think I'm pretty much just going to keep stage-3 projects around (with low dex as applicable), and assign them `pass` and `disregard` tags as applicable if and when they become the sort of thing I'd pass on, or disregard.
